{"success": true, "data": [{"team": {"name": "Error418", "id": 348}, "date": "2023-08-26T01:47:40.588468+00:00", "challenge_id": 1, "challenge": {"name": "\u25b6 Sanity Check", "category": "Misc", "value": 39, "id": 1}, "user": {"name": "Strike", "id": 1968}, "type": "correct", "id": 3765}], "meta": {"count": 1}}
